.github/workflows/deploy.yml

```diff
@@ -34,6 +35,33 @@ jobs:
             --force-publish \
             --yes
 
+  relase-candidate:
+    runs-on: ubuntu-latest
+    if: github.ref_name == 'release' && github.repository_owner == 'dequelabs'
```

Comment:
With the date appended this will never return true. Will need to update to handle dates in the branch name